tests/integration/team_test.py

```diff
@@ -17,34 +17,88 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
+
 import pytest
 
 import libnmstate
-from libnmstate.error import NmstateNotImplementedError
+from libnmstate.error import NmstateLibnmError
 from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Team
 
+from .testlib import assertlib
+from .testlib import cmd as libcmd
+
+
+DNF_REMOVE_NM_TEAM_CMD = ("dnf", "remove", "-y", "-q", "NetworkManager-team")
```

Comment:
When I run the test in my VM, since nmstate will lose default gateway(old long story), the `dnf` command will fail to
install the package back.

Can you try to move the so file to /tmp and move it back after done?

```
mv `rpm -ql NetworkManager-team |egrep so$` /tmp
```

Reply:
This is not working. If I do that the test is not failing.

Reply:
Done, thanks!